src/CMakeLists.txt

```diff
@@ -17,6 +17,9 @@ target_link_libraries(${PROJECT_LIBRARY_TARGET_NAME}
 target_include_directories(${PROJECT_LIBRARY_TARGET_NAME} PRIVATE
   ${ignition-math${IGN_MATH_VER}_INCLUDE_DIRS})
 
+target_include_directories(${PROJECT_LIBRARY_TARGET_NAME} PUBLIC
+  ${ignition-utils${IGN_UTILS_VER}_INCLUDE_DIRS})
```

Comment:
this means ign-utils1 is now required by the core `ignition-common4` library. we should update the find_package call to make this dependency `REQUIRED`